tests/common/test_version.py

```diff
@@ -115,6 +116,10 @@ def test_distro_is_correct_for_exception_case(self, *args): # pylint: disable=us
         self.assertListEqual(default_list, osinfo)
         return
 
+    def test_lis_version(self, *args): # pylint: disable=useless-return,unused-argument
```

Comment:
please address the pylint warnings (remove the return statement and use *_ instead of *_args)

Reply:
This is consistent with all the above unit tests. Do you want those changed too?

Reply:
We actually recently enabled the linters on our repo. The above tests haven't been touched in a while so we have it commented, but we will eventually fix them. All the new code though is supposed to be without any linter errors so it would be great if you could address these errors! :)

Reply:
Ok, since it wouldn't make sense to have just one new unit test dissimilar to all the others, I went ahead and fixed the trivial pylint errors in this file.

Reply:
Awesome thanks for that :)